debian_packages.lst

```diff
@@ -1,6 +1,5 @@
 git
 libffi-dev
-libmysqlclient-dev
```

Comment:
@viadanna is there a reason this was removed from the package list? It breaks devstack provisioning for me, because one of the python requirements requires it.